mne/report.py

```diff
@@ -1128,13 +1220,12 @@ def add_bem_to_section(self, subject, caption='BEM', section='bem',
                                 caption=caption)
         html, caption, _ = self._validate_input(html, caption, section)
         sectionvar = self._sectionvars[section]
-
-        self.fnames.append('%s-#-%s-#-custom' % (caption[0], sectionvar))
-        self._sectionlabels.append(sectionvar)
-        self.html.extend(html)
+        self._add_or_replace('%s-#-%s-#-custom' % (caption[0], sectionvar),
+                             sectionvar, html)
```

Comment:
FYI @wmvanvliet unfortunately this was not backward compatible. It would have been if the original call was `self.html.append` but it was `.extend`. Fix coming